organize_content/exercise2-organize-content.md

```diff
@@ -49,20 +55,10 @@ If you do not have an existing SSH key pair, generate a new one:
 A public and private key are generated. [Add the public SSH key to your GitLab account](#add-an-ssh-key-to-your-gitlab-account)
 and keep the private key secure.
 
-You really shouldn't use a DSA or ECDSA key type because they're no longer considered secure.
 
 NOTE:
 Treat a private key like a password. Keep it in a safe place. Don't let anyone know its value.
 
-To use SSH to communicate with GitLab, you need:
-
-- The OpenSSH client, which comes pre-installed on GNU/Linux, macOS, and Windows 10.
-- SSH version 6.5 or later. Earlier versions used an MD5 signature, which is not secure.
-
-Remember that every time the Git client communicates with GitLab, it needs to provide the private key as proof of your identity. So you need to always keep the private key on your computer.
-
-To view the version of SSH installed on your system, run `ssh -V`.
-
 To communicate with GitLab, you can use the following SSH key types:
```

Comment:
**Comment** @Preshh0 I suggest moving this to before the task, because it's important that the user knows this **before** creating a key pair.